build.bat

```diff
@@ -146,15 +146,15 @@ exit /b
 
 :build_data
 rem call :build_essay
+if not defined BASH_DIR (
+  set BASH_DIR=C:\Program Files\Git\usr\bin
+)
+set PATH=%PATH%;%BASH_DIR%
 copy %WEASEL_ROOT%\LICENSE.txt output\
 copy %WEASEL_ROOT%\README.md output\README.txt
-copy %WEASEL_ROOT%\brise\essay.txt output\data\
-copy %WEASEL_ROOT%\brise\default.yaml output\data\
-copy %WEASEL_ROOT%\brise\symbols.yaml output\data\
-copy %WEASEL_ROOT%\brise\preset\*.yaml output\data\
-copy %WEASEL_ROOT%\brise\supplement\*.yaml output\data\
-if not exist output\expansion mkdir output\expansion
-copy %WEASEL_ROOT%\brise\extra\*.yaml output\expansion\
+cd %WEASEL_ROOT%\brise
+bash -c "export brise_dir=. rime_dir=../output/data && bash rime-install"

```

Comment:
Optional: `rime-install` can be called from outside `brise` directory, so we can omit the `cd` commands before and after.